.github/workflows/build_quic_interop_container.yml

```diff
@@ -38,7 +38,12 @@ jobs:
           docker login -u openssl-ci+machine -p ${{ secrets.QUAY_IO_PASSWORD }} quay.io
       - name: Patch qns.Dockerfile
         run: |
-          sed 's/RUN     cmake -DQUIC_BUILD_TOOLS=on -DQUIC_ENABLE_LOGGING=on ../RUN     cmake -DQUIC_BUILD_TOOLS=on -DQUIC_ENABLE_LOGGING=on -DQUIC_TLS_LIB=openssl ../' ./scripts/qns.Dockerfile
+          sed -i 's/RUN     cmake -DQUIC_BUILD_TOOLS=on -DQUIC_ENABLE_LOGGING=on ../RUN     cmake -DQUIC_BUILD_TOOLS=on -DQUIC_ENABLE_LOGGING=on -DQUIC_TLS_LIB=openssl ../' ./scripts/qns.Dockerfile
+          if grep -q "RUN     cmake -DQUIC_BUILD_TOOLS=on -DQUIC_ENABLE_LOGGING=on -DQUIC_TLS_LIB=openssl .." ./scripts/qns.Dockerfile; then echo "Patched successfully"; else exit 1; fi
+      - name: Patch .dockerignore
+        run: |
+          echo "!submodules/openssl" >> .dockerignore
+          cat .dockerignore
```

Comment:
crud, sorry about this, I should have added this to the dockerignore file in the msquic repo.  If you wanted to go ahead and submit a PR for that to the msquic repo, I would appreciate it.